apps/todomvc-remix/app/components/Todos.tsx

```diff
@@ -0,0 +1,43 @@
+import { SerializeFrom } from "@remix-run/node";
+import { Todo } from "keelClient";
+
+type Props = {
+  todos: SerializeFrom<Todo[]>;
+};
+export const Todos = ({ todos }: Props) => {
+  const checkTodo = (id: string, checked: "true" | "false") => {
+    fetch("/?index", {
+      method: "POST",
+      body: new URLSearchParams({
+        action: "update",
+        checked,
+        id,
+      }),
+    });
+  };
+  // @todo error handling
+
+  return (
+    <ul>
+      {todos.map((t) => (
+        <li key={t.id}>
+          <label>
+            <input
```

Comment:
I think a slightly more remix-y way of doing this would be to just use a form and have an onchange that submits it. By default a submitted form will hit the route action so you don't need to do a manual fetch

Reply:
Great shout! Let's do it like that!